tests/integration/galley/dialout_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package galley
+
+import (
+	"fmt"
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/galley"
+	"istio.io/istio/pkg/test/framework/components/mcpserver"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"istio.io/istio/pkg/test/util/retry"
+	"istio.io/istio/pkg/test/util/structpath"
+)
+
+var yamlCfg = `
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: helloworld-gateway
+spec:
+  selector:
+    istio: ingressgateway # use istio default controller
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - "*"
+`
+
+func TestDialout_Basic(t *testing.T) {
+	// TODO: Limit to Native environment until the Kubernetes environment is supported in the Galley
```

Comment:
Can any of these tests be run in parallel? If they can, let's mark them so ... if we decide to make separate targets for each suite, we can take advantage of parallelism.

Reply:
Possibly, but unlikely. For the native case, I'd assume there would be port clashes. The only K8s one is validation, and that should be able to run in parallel, but there is no other K8s test active at the moment. So marking them parallel at this point won't save much.

I'd prefer to that as a separate work, after making all of these environment agnostic first. I can do the parallelization changes as part of that.